airflow/providers/apprise/notifications/apprise.py

```diff
@@ -52,9 +51,9 @@ def __init__(
         *,
         body: str,
         title: str | None = None,
-        notify_type: NotifyType | None = None,
-        body_format: NotifyFormat | None = None,
-        tag: str | Iterable[str] | None = None,
+        notify_type: NotifyType = NotifyType.INFO,
+        body_format: NotifyFormat = NotifyFormat.TEXT,
+        tag: str | Iterable[str] = "all",
```

Comment:
It seems like a breaking change - theoretically, someone could init. the class with either `notify_type=None, body_format=None, tag=None` and it will fail.
I suggest deprecating it gradually, i.e. - if any of the params is `None` - raise a deprecation warning + assign the corresponding default).

Reply:
Great solution!